### ISSUE TITLE
Make test fails without error messages

### ISSUE THREAD

**Issue by jhossbach** [espressomd/espresso#4370]
MWE: Clone latest commit from python branch; run `cmake .. && make && make test`
Some tests will fail without error messages

**Comment by jngrad** [comment]
Running `env CTEST_OUTPUT_ON_FAILURE=1 make test` reveals the error messages:
```
  4/361 Test   #4: test_checkpoint_lb.cpu-p3m.cpu-lj-therm.lb_0 .................***Failed    0.93 sec
Traceback (most recent call last):
  File "/work/jgrad/espresso-fork-PR/build-test/testsuite/python/test_checkpoint_lb.cpu-p3m.cpu-lj-therm.lb_0.py", line 20, in <module>
    import unittest_decorators as utx
ModuleNotFoundError: No module named 'unittest_decorators'
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
mpiexec detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[13597,1],0]
  Exit code:    1
--------------------------------------------------------------------------
```

The `testsuite/python` folder contains files like `unittest_decorators.py` and `tests_common.py` that need to be copied over by `make python_test_data`, which is a dependency of `make check_python`, but it's currently not a dependency of `make test`.

**Comment by jngrad** [comment]
I've just tried adding `set_tests_properties(${TEST_NAME} PROPERTIES DEPENDS "pypresso;check_python")` to the python tests but it changed nothing. This a bug in CMake that has been open for 7 years and still hasn't been solved ([cmake/cmake#8774](https://gitlab.kitware.com/cmake/cmake/-/issues/8774) and [ticket 8774](https://public.kitware.com/Bug/view.php?id=8774)). There is a popular workaround ([StackOverflow 736838](https://stackoverflow.com/a/736838)) that consists in defining a CMake target `check` that does the same thing as the `test` pseudo-target (it's not a real CMake target: one cannot set its target properties, such as dependencies). Contrary to `test`, `check` is a real target that understands dependencies. We already use this workaround in ESPResSo.
